src/ProjectWindow.cpp

```diff
@@ -708,30 +706,6 @@ std::pair<int, int> ProjectWindow::ViewportSize() const
    return { width, height };
 }
 
-bool ProjectWindow::MayScrollBeyondZero() const
-{
-   auto pProject = FindProject();
-   if (!pProject)
-      return false;
-   auto &project = *pProject;
-   auto &scrubber = Scrubber::Get( project );
```

Comment:
This deletion also allows you to eliminate
`#include "Scrubbing.h"`
at the top of the file.  Please do that!